components/donate-button.jsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import {FormattedMessage, IntlMixin, FormattedNumber} from 'react-intl';
+
+module.exports = React.createClass({
```

Comment:
actually yeah I think adding it doesn't hurt here since it is optional proptype anyway. I will let you decide here then.